KubeArmor/BPF/system_monitor.c

```diff
@@ -275,6 +282,12 @@ typedef struct __attribute__((__packed__)) sys_context
 #define BPF_PERF_OUTPUT(_name) \
     BPF_MAP(_name, BPF_MAP_TYPE_PERF_EVENT_ARRAY, int, __u32, 1024)
 
+#define BPF_RINGBUF(_name, _size) \
```

Comment:
usually BPF rigbuffer is supported on kernels >= 5.8, so it will break system monitor on older kernels.